### PR TITLE
[Delivers #99273060] Send cancellation conf email to active observers

### PR DESCRIPTION
**Pull request by jessieay**
* Active observers are observer who are not approver or requester
* Previously, were not receiving an email
* Story: https://www.pivotaltracker.com/story/show/99273060

### CODE REVIEW

**Comment by pkarman** [comment]
I will QA

**Comment by pkarman** [comment]
I left QA comments on the PT story. Looks great.

File: app/models/user.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@ class User < ActiveRecord::Base
 
   has_many :approvals
   has_many :observations
+  has_many :observers, through: :observers, source: :user
   has_many :comments
```
Comment (pkarman) [inline app/models/user.rb]: why this recursive relationship definition?

**Comment by pkarman** [comment]
@jessieay - just one review comment, and then I'm ready to :sheep: it

File: app/models/user.rb (hunk 1 above)
Comment (jessieay) [inline app/models/user.rb]: oops! meant for this to be `through: :observations`...wonder how that was even working!?

**Comment by jessieay** [comment]
@pkarman fixed!

**Pull request merged**